utils/doclint/generateDotnetApi.js

```diff
@@ -0,0 +1,333 @@
+
+  return node;
+}
+
+/**
+ * @param {string} text
+ * @param {number=} maxColumns
+ * @param {string=} prefix
+ */
+function wrapText(text, maxColumns = 0, prefix = '') {
+  if (!maxColumns) {
+    return prefix + text;
+  }
+
+  // we'll apply some fixes, like rendering the links from markdown
+  // TODO: maybe we can move this to either markdown.js or documentation.js?
+  text = text.replace(/\[([^\[\]]*)\]\((.*?)\)/g, function (match, name, url) {
+    return `<a href="${url}">${name}</a>`;
+  });
+
+  const lines = [];
+
+  let insideTag = false;
+  let escapeChar = false;
+  let insideLink = false;
+  let breakOnSpace = false;
+  let insideNode = false;
+
+  let line = "";
+  let currentWidth = 0;
+
+  let prevChar = '';
+  for (let i = 0; i < text.length; i++) {
+    const char = text.charAt(i);
+    let skipThisChar = true;
+
+    if (['<', '['].includes(char)) {
+      // maybe we should break if a node starts, and we're almost at the end of the block
+      if (!insideNode && currentWidth >= maxColumns * 0.85) {
+        lines.push(line);
+        line = "";
+        currentWidth = 0;
+      }
+      insideNode = true;
+      insideTag = true;
+    } else if (['>', ']'].includes(char)) {
+      insideTag = false;
+      if (prevChar === '/') { // self-closing tag
+        insideNode = false;
+      }
+    } else if (char === '/' && prevChar === '<') { // closing node
+      insideNode = false;
+    } else if (char === '(' && prevChar === ']') {
+      insideLink = true;
+    } else if (char === ')' && insideLink) {
+      insideLink = false;
+    } else if (char === `\\`) {
+      escapeChar = true;
+    } else if (char === " " && breakOnSpace) {
+      breakOnSpace = false;
+      lines.push(line);
+      line = "";
+      currentWidth = 0;
+      continue;
+    } else {
+      skipThisChar = false;
+    }
+
+    if (currentWidth == 0 && char === " ") {
+      continue;
+    }
+
+    line += char;
+    currentWidth++;
+
+    prevChar = char;
+    if (skipThisChar) {
+      continue;
+    }
+
+    if (currentWidth >= maxColumns
+      && !insideTag
+      && !escapeChar
+      && !insideLink
+      && !insideNode) {
+      breakOnSpace = true;
+    }
+  }
+
+  // make sure we push the last line, if it hasn't been pushed yet
+  if (line !== "") {
+    lines.push(line);
+  }
+
+  return lines;
+}
+
+/**
+ * @param {string} memberKind  
+ * @param {string} name 
+ * @param {Documentation.Member} member */
+function translateMemberName(memberKind, name, member) {
+
+  // check if there's an alias in the docs, in which case
+  // we return that, otherwise, we apply our dotnet magic to it
+  if (member) {
+    if (member.alias !== name) {
+      return member.alias;
+    }
+  }
+
+  let assumedName = name.charAt(0).toUpperCase() + name.substring(1);
+  switch (memberKind) {
+    case "interface":
+      return `I${assumedName}`;
+    case "method":
+      return `${assumedName}Async`;
+    case "event":
+      return `On${assumedName}`;
```

Comment:
This will be quite breaking 🤓

Reply:
I agree. I'll get rid of it and keep using the _actual_ name from the docs.